examples/jsonpatch-yaml.md

```diff
@@ -0,0 +1,81 @@
+# Demo: applying a json patch
+
+This is the same example as in [jsonpatch.md](jsonpatch.md), but using YAML as format
+for the JSON6902 patches.
+
+The example below modifies an `Ingress` object with such a patch.
+
+Make a `kustomization` containing an ingress resource.
+
+<!-- @createIngress @test -->
+```
+DEMO_HOME=$(mktemp -d)
+
+cat <<EOF >$DEMO_HOME/kustomization.yaml
+resources:
+- ingress.yaml
+EOF
+
+cat <<EOF >$DEMO_HOME/ingress.yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: my-ingress
+spec:
+  rules:
+  - host: foo.bar.com
+    http:
+      paths:
+      - backend:
+          serviceName: my-api
+          servicePort: 80
+EOF
+```
+
+Declare a JSON patch file to update two fields of the Ingress object:
+
+- change host from `foo.bar.com` to `foo.bar.io`
+- change servicePort from `80` to `8080`
+
+<!-- @addJsonPatch @test -->
+```
+cat <<EOF >$DEMO_HOME/ingress_patch.yaml
```

Comment:
I don't think we need to add a second file for the json patch example. Maybe you can add this patch to `jsonpatch.md` with the explanation that Kustomize also supports yaml format for json patch.

Reply:
Yes, agreed. That would be definitely nicer. I'll redo it that way.